### PR TITLE
Admin privileges

### PR DESCRIPTION
**Pull request by hailexiao**
Added the ability for admin users to delete non-admin users, reviews, and Yelpers

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'pry'
```
Comment (EliseFitz15) [inline app/controllers/reviews_controller.rb]: remove if you are not using pry... also if you have pry-rails gem you should not need to require on individual files like sinatra.

File: spec/features/admin_deletes_user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    fill_in 'Email', with: admin.email
+    fill_in 'Password', with: admin.password
+
+    click_button 'Log in'
+
+    visit user_path(user)
+
+    click_button 'Delete User'
+
+    expect(page).to have_content('User deleted.')
```
Comment (EliseFitz15) [inline spec/features/admin_deletes_user_spec.rb:34]: You will want to make sure all associated objects are also deleted.

File: spec/features/admin_deletes_yelper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    fill_in 'Email', with: admin.email
+    fill_in 'Password', with: admin.password
+
+    click_button 'Log in'
+
+    visit yelper_path(yelper)
+
+    click_button 'Delete Yelper'
+
+    expect(page).to have_content('Yelper deleted.')
```
Comment (EliseFitz15) [inline spec/features/admin_deletes_yelper_spec.rb:27]: Same as above you will want to be sure all associated objects (i.e. reviews) are deleted as well.

Comment (hailexiao) [inline spec/features/admin_deletes_yelper_spec.rb:27]: If the Yelper no longer exists, there is no way for a user to view the associated reviews and votes. Should we check the database?

**Pull request merged**